core/src/main/java/org/apache/druid/indexer/partitions/SingleDimensionPartitionsSpec.java

```diff
@@ -39,10 +39,12 @@
  */
 public class SingleDimensionPartitionsSpec implements DimensionBasedPartitionsSpec
 {
-  static final String NAME = "single_dim";
+  public static final String NAME = "single_dim";
   static final String OLD_NAME = "dimension";  // for backward compatibility
 
+  private static final String PARITION_DIMENSION = "partitionDimension";
   private static final String MAX_PARTITION_SIZE = "maxPartitionSize";
+  private static final String FORCE_GUARANTEED_ROLLUP_COMPATIBLE = "";
```

Comment:
nit: unused

Reply:
It's used in my follow-up PR that adds the implementation of range partitioning.